collectors/framework/api.py

```diff
@@ -74,7 +74,7 @@ class status(APIView):
                                 },
                                 "error": {"type": "object", "nullable": True},
                                 "is_complete": {"type": "boolean"},
-                                "is_up2date": {"type": "bool"},
+                                "is_up2date": {"type": "boolean"},
```

Comment:
So this is on me :smile: how could I put `boolean` and `bool` on the very next line ...

Reply:
It happens :smile: